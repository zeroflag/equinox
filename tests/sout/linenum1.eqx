\ throw an error
: nested3
  ( errror comes here)
  nil "invalid concat" .. ;

: nested2 nested3 ;
: nested1 nested2 ;

: babbage
  1
  begin
    1 +
    \ "xx" error/1
    dup dup *
    nested1
    1000000 %
    \ call the faulty word
    nested3
    269696 =
  until ;

babbage
