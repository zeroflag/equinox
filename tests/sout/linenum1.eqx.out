Error occurred at line: 4 (tests/sout/linenum1.eqx)
  002.  : nested3
  003.    ( errror comes here)
=>004.    nil "invalid concat" .. ;
  005.  
  006.  : nested2 nested3 ;

